Gruntfile.js

```diff
@@ -324,40 +324,29 @@ module.exports = function(grunt) {
     },
 
     /**
-     * JSHint: https://github.com/gruntjs/grunt-contrib-jshint
-     *
-     * Validate files with JSHint.
-     * Below are options that conform to idiomatic.js standards.
-     * Feel free to add/remove your favorites: http://www.jshint.com/docs/#options
+     * Lint the JavaScript.
      */
-    jshint: {
-      options: {
-        camelcase: false,
-        curly: true,
-        forin: true,
-        immed: true,
-        latedef: true,
-        newcap: true,
-        noarg: true,
-        quotmark: true,
-        sub: true,
-        boss: true,
-        strict: true,
-        evil: true,
-        eqnull: true,
-        browser: true,
-        plusplus: false,
-        globals: {
-          jQuery: true,
-          $: true,
-          module: true,
-          require: true,
-          define: true,
-          console: true,
-          EventEmitter: true
-        }
-      },
-      all: ['static/js/main.js']
+    lintjs: {
+      /**
+       * Validate files with ESLint.
+       * https://www.npmjs.com/package/grunt-contrib-eslint
+       */
+      eslint: {
+        options: {
+            config: "_settings/eslint.yaml",
+            quiet: false
+        },
+        src: [
+          'static/js/jquery.cf_input-split.js',
+          'static/js/jquery.cf_pagination.js',
+          'static/js/jquery.custom-input.js',
+          'static/js/jquery.custom-select.js',
+          'static/js/jquery.type-and-filter.js',
+          'static/js/breakpoint-handler.js',
+          'static/js/content-slider.js',
+          'static/js/app.js'
+        ]
```

Comment:
Could this be reduced to

```
src: [
    'static/js/**/*',
    '!static/js/main.js',
    '!static/js/main.min.js'
]
```
That way if a new file is added the gruntfile doesn't need to be updated?